<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChoiceDialog</class>
 <widget class="QDialog" name="ChoiceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose type of payment</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="2">
      <widget class="QGroupBox" name="percentGroupBox">
       <property name="title">
        <string>Percentages</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QPushButton" name="plusConstButton">
            <property name="text">
             <string>++</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="plusButton">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QDoubleSpinBox" name="percentDoubleSpinBox">
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="lessButton">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="lessConstButton">
            <property name="text">
             <string>--</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="0" rowspan="4">
      <widget class="QGroupBox" name="paramsGroupBox">
       <property name="title">
        <string>Parameters</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="typeGroupBox">
       <property name="title">
        <string>Type of payment</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QRadioButton" name="cashButton">
            <property name="text">
             <string>Cash</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="checkButton">
            <property name="text">
             <string>Check</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="visaButton">
            <property name="text">
             <string>Credit Card</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="otherButton">
            <property name="text">
             <string>Other</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QRadioButton" name="bankingButton">
            <property name="text">
             <string>Banking</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QRadioButton" name="dueButton">
            <property name="text">
             <string>Due</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QGroupBox" name="freeBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>75</height>
        </size>
       </property>
       <property name="title">
        <string>Free Values</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>271</width>
          <height>54</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="0" column="0">
          <widget class="QLabel" name="freeLabel">
           <property name="text">
            <string>Text</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="freeEdit"/>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="freeValueSpinBox">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="freeValueLabel">
           <property name="text">
            <string>Value</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="quitButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>&amp;Quit and cancel</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="okButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Choose and &amp;Ok</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QGroupBox" name="distanceGroupBox">
       <property name="title">
        <string>Round Trip</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="QDoubleSpinBox" name="distanceDoubleSpinBox"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cashButton</tabstop>
  <tabstop>checkButton</tabstop>
  <tabstop>visaButton</tabstop>
  <tabstop>otherButton</tabstop>
  <tabstop>bankingButton</tabstop>
  <tabstop>dueButton</tabstop>
  <tabstop>plusButton</tabstop>
  <tabstop>percentDoubleSpinBox</tabstop>
  <tabstop>lessButton</tabstop>
  <tabstop>lessConstButton</tabstop>
  <tabstop>plusConstButton</tabstop>
  <tabstop>freeEdit</tabstop>
  <tabstop>freeValueSpinBox</tabstop>
  <tabstop>quitButton</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>distanceDoubleSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
